src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -280,6 +280,11 @@ private boolean setUpLogging(ContestConfig config) {
           Logger.log(Level.INFO, "Reading CDF cast vote record file: %s...", cvrPath);
           new CommonDataFormatReader(cvrPath, config).parseCvrFile(castVoteRecords);
           continue;
+        } else if (ContestConfig.getProvider(source) == Provider.CLEAR_BALLOT) {
+          ClearBallotCvrReader reader = new ClearBallotCvrReader(cvrPath, config);
+          Logger.log(Level.INFO, "Reading Clear Ballot cast vote records from folder: %s...", cvrPath);
```

Comment:
It looks like Clear Ballot is (blessedly) a single .csv file, so why do we specify a folder instead of a specific file like for ES&S? The latter would seem to make more sense to me...